marketplace/README.md

```
@@ -1,13 +1,34 @@
 ```bash
 claude plugin marketplace add https://github.com/m4ttstack/mattstack-marketplace
-claude plugin install mattstack@mattstack
-claude plugin install fast-browser@mattstack
 ```
 
+## Usage
+
+Install any plugin from the catalog:
+
+```console
+$ claude plugin install mattstack@mattstack      # the mattstack skills pack
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)